rest-server/src/models/job.js

```diff
@@ -92,81 +68,32 @@ class Job {
           });
           jobList.sort((a, b) => b.createdTime - a.createdTime);
           next(jobList);
-        });
+        } catch (error) {
```

Comment:
I suggest we refine error handling using a promise library like [bluebird](https://github.com/petkaantonov/bluebird) later, replacing callbacks with `Promise`.

Reply:
Good suggestion. Put all these work about code refinement in backlog?